tests/unit/test_uploader.py

```diff
@@ -8,7 +8,7 @@
     SCCUploader
 )
 
-FakeResponse = namedtuple('FakeResponse', 'status_code')
+FakeResponse = namedtuple('FakeResponse', 'status_code, headers')
```

Comment:
Note that we could change this to 
```suggestion
class FakeResponse:
    def __init__(self, status_code, headers=None):
        if headers is None:
            headers = {}
        self.status_code = status_code
        self.headers = headers
```
And then you wouldn't need to change any of the FakeResponse calls in the rest of the code unless you wanted to include header fields.

Reply:
updated with the suggested changes